libs/service-portal/finance/src/components/DocumentScreen/DocumentScreen.tsx

```diff
@@ -162,10 +168,11 @@ const DocumentScreen: FC<Props> = ({
               labelClearAll={formatMessage(m.clearAllFilters)}
               labelOpen={formatMessage(m.openFilter)}
               labelClose={formatMessage(m.closeFilter)}
+              popoverFlip={false}
```

Comment:
Just curious, what does this do? Prevent the calendar from popping up or down? 🤔

Reply:
The default state is popping down. So it will prevent it from popping up.

Reply:
changed the code comment a bit to reflect more on what this does.